README.md

```diff
@@ -1,87 +1,231 @@
-# Unified-Language-Server
+# unified-language-server
```

Comment:
I copied the readme from `rehype-format` and adjusted it to match the new readme format used throughout unified.